### DISCUSSION TITLE
Welcome to Qiskit Aer Discussions!

### DISCUSSION THREAD

**Discussion by doichanj** [Qiskit/qiskit-aer#2068] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tim-Li** [comment]
Hi~
I try to use qiskit-aer mpi on a supercomputer with 552 nodes and each node has 512G memory. (here are some initial test results：https://github.com/Tim-Li/nchc-hpc_qc-bechmark ) 
Do you have any recommendations for some interesting benchmarks?

And I also try to build a 30 nodes nvidia-jason-nano cluster for mpi quantum circuit simulation. Do you think this idea is feasible?
I think it's well-suited to be a tutorial demonstration for qiskit-aer on multi-GPU and multi-node setups.
